georocket-server/src/main/java/io/georocket/http/Endpoint.java

```diff
@@ -12,4 +21,63 @@
    * @return the router
    */
   Router createRouter();
+
+  /**
+   * Get absolute data store path from request
+   * @param context the current routing context
+   * @return the absolute path (never null, default: "/")
+   */
+  static String getEndpointPath(RoutingContext context) {
```

Comment:
Please put these new static methods in a separate abstract class named `AbstractEndpoint`